### PR TITLE
PLANET-3483: EN Form Block: Add new side style in the front end

### PR DESCRIPTION
**Pull request by pablocubico**
Ref: https://jira.greenpeace.org/browse/PLANET-3483

### CODE REVIEW

**Comment by pablocubico** [comment]
Current looks: 
image

image

File: classes/controller/menu/class-enform-post-controller.php [hunk 1]
```diff
@@ -177,6 +177,26 @@ public function handle_form_shortcode( $atts ) {
...
+					'last'        => 'Last name',
+					'address1'    => 'Address 1',
+					'address2'    => 'Address 2',
+					'city'        => 'City',
+					'country'     => 'Country',
+					'post'        => 'Postcode',
+					'email'       => 'Email',
+					'region'      => 'County',
+					'phoneNumber' => 'Phone Number',
+					'birth'       => 'birthday'
```
Comment (dpivo) [inline classes/controller/menu/class-enform-post-controller.php]: phpcs error: every line must end in a comma
```suggestion
					'birth'       => 'birthday',
```

**Comment by angtheod** [comment]
When we add required fields they appear in the `Form preview` box and they prevent post from Publish/Update unless they get filled. Can we make this to not happen within the post?

image

**Comment by kirdia** [comment]
this should be handled by this. It doesn't work? can you check if the newest js is loaded?
https://github.com/greenpeace/planet4-plugin-engagingnetworks/pull/103/files#diff-238455a84a6d8a49849a0f6ad4b932c7R41

**Comment by angtheod** [comment]
> this should be handled by this. It doesn't work? can you check if the newest js is loaded?
> https://github.com/greenpeace/planet4-plugin-engagingnetworks/pull/103/files#diff-238455a84a6d8a49849a0f6ad4b932c7R41

You are right, JS wasn't loaded. Version should be updated [here](https://github.com/greenpeace/planet4-plugin-engagingnetworks/pull/103/files#diff-3118d8274643fd4814e1f19d0b0c89d3R367)

File: admin/js/enforms.js [hunk 2]
```diff
@@ -35,6 +36,11 @@ jQuery(function ($) {
     $('#p4enform_fields').val(JSON.stringify(p4_enform.fields.toJSON()));
   });
 
+  /**
+   * Disable preview form fields.
+   */
+  $('#meta-box-form :input').prop("disabled", true);
```
Comment (angtheod) [inline admin/js/enforms.js]: Please use single quotes for strings, so that ESlint does not complain (also in lines 207, 431)

**Comment by angtheod** [comment]
Few minors that ESlint complains but are not in PR:
[no-unused-vars](https://github.com/greenpeace/planet4-plugin-engagingnetworks/pull/103/files#diff-238455a84a6d8a49849a0f6ad4b932c7R35), [no-redeclare](https://github.com/greenpeace/planet4-plugin-engagingnetworks/pull/103/files#diff-238455a84a6d8a49849a0f6ad4b932c7R241), [semi](https://github.com/greenpeace/planet4-plugin-engagingnetworks/pull/103/files#diff-238455a84a6d8a49849a0f6ad4b932c7R411)

File: includes/enblock.twig [hunk 3]
```diff
@@ -14,29 +14,48 @@
 		<div class="container">
 			<div class="row">
 				<div class="col-md-12">
+
+					{% if fields.en_form_style == 'side-style' %}
+						<div class="form-caption">
+							<h1>{{ fields.left_title }}</h2>
```
Comment (angtheod) [inline includes/enblock.twig]: closing tag mismatch

Comment (pablocubico) [reply includes/enblock.twig]: Fixed.

**Comment by pablocubico** [comment]
@angtheod branch updated, ESLint warnings fixed, tag mismatch fixed!

**Pull request merged**